core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+
+form {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    justify-content: center;
+}
+
+.ai-image-prompt__input {
+    display: flex;
+    flex-direction: column;
+
+    > div {
+        display: flex;
+        align-items: center;
+    }
+
+    > span {
+        display: flex;
+        margin: $spacing-0 0 $spacing-1 $spacing-5;
+        font-size: $font-size-sm;
+        color: $color-palette-gray-700;
+    }
+}
+
+.ai-image-prompt__text {
+    .field:last-child {
+        margin-bottom: 0;
+    }
+
+    .ai-image-prompt__textarea {
+        height: 365px;
+    }
+
+    &.ai-image-prompt__text--generated {
+        .ai-image-prompt__textarea {
+            height: 90px;
+        }
+
+        .ai-image-prompt__rewritten-textarea {
+            height: 230px;
```

Comment:
parse this pixels to rem. If we change the root font size all the sizes written in pixels will probably break